src/main/webapp/college-info.html

```diff
@@ -8,11 +8,11 @@
   </head>
   <body>
     <header>
-      <h1>Black Student Insight</h1>
-      <input type="text" id="school-search">
-      <search-button id="search-button">Search</search-button>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button id="search-button" onclick="loadSchoolInfo()">Search</button>
```

Comment:
You may need to merge with Toba's change (or maybe his isn't in yet...) that removes these click handlers from the HTML Double check that.